### DISCUSSION TITLE
ffmpeg -loglevel error

### DISCUSSION THREAD

**Discussion by InstantRecruiting** [scivision/PyLivestream#97] [Q&A]
``ffmpeg -loglevel error -y -re -stream_loop -1 -i video.mp4 -codec:v libx264 -pix_fmt yuv420p -preset veryfast -b:v 13000k -g 59.94005994005994 -codec:a aac -b:a 128k -ar 44100 -maxrate 13000k -bufsize 6500k -strict experimental -f flv``
Operating system (Windows)
I'm getting this error, anyone help me.

**Comment by scivision** [comment]
I don't see the error message posted here. It seems possible to be an issue with FFmpeg itself--try the command line above without using PyLivestream.

**Comment by the-iconic-rihan** [comment]
ffmpeg -loglevel error -f x11grab -s 1280x720 -i :0.0+50,30 -f pulse -i default -codec:v libx264 -pix_fmt yuv420p -preset veryfast -b:v 2500k -g 120 -codec:a aac -b:a 128k -ar 44100 -maxrate 2500k -bufsize 1250k -strict experimental -f flv rtmp://a.rtmp.youtube.com/live2/streamkey

Even im facing the same issue , for webcamstream, screensharelivestream. Along with im facing 'av_interleaved_write_frame(): Broken pipe' error
Anyone any solution?

**Comment by scivision** [comment]
I'm not sure. It could be an issue with the particular FFmpeg version. Perhaps try on another computer.
